scanomatic/image_analysis/first_pass_image.py

```diff
@@ -36,8 +38,35 @@ def get_image_scale(im):
         return invalid_scale
 
 
-def _get_rotated_vector(x, y, rotation):
-    return x * np.cos(rotation), y * np.sin(rotation)
+# def _get_rotated_vector(x, y, rotation):
```

Comment:
this was only used in commented out code so will keep until commented out code is removed

Reply:
There is an issue for this, right?

Reply:
Now there is https://github.com/Scan-o-Matic/scanomatic-standalone/issues/97